### PR TITLE
Site: exclude README.md and .gitignore [ci skip]

### PR DESCRIPTION
**Pull request by crispgm**
😄

### CODE REVIEW

**Comment by envygeeks** [comment]
I see no reason not to.  LGTM.

File: site/_config.yml [hunk 1]
```diff
@@ -32,3 +32,7 @@ gems:
   - jekyll-seo-tag
   - jekyll-avatar
   - jekyll-mentions
+
+exclude:
+  - README.md
+  - .gitignore
```
Comment (parkr) [inline site/_config.yml]: Interesting! This should be removed automatically.

**Comment by parkr** [comment]
LGTM.

File: site/_config.yml (hunk 1 above)
Comment (crispgm) [inline site/_config.yml]: @parkr if these files were generated with `jekyll new` by default, we could add them to template of `_config.yml`. Repos with jekyll site have large possibilities contain such files as `README.md`, `LICENSE`, `Rakefile`, `.gitignore`, etc.

Comment (ashmaroli) [inline site/_config.yml]: @crispgm, currently the files [here](https://github.com/jekyll/jekyll/tree/master/lib/site_template) plua a Gemfile are copied to a new site's root by `jekyll new`

Comment (ashmaroli) [inline site/_config.yml]: so, is there really a need to have this exclusion, by default?

**Comment by envygeeks** [comment]
@jekyllbot: merge +site

**Comment by ashmaroli** [comment]
@DirtyF, I cant find the documentation that explains the principle behind the **exclude array** in a _config.yml. Please help.

**Comment by envygeeks** [comment]
@ashmaroli the exclude array prevents a file from being written.  So say you don't want your Gemfile, or README.md or vendor/ folder to end up being processed, you exclude them and they aren't processed or written. On a side note: I just seen your comment, sorry I merged before we talked about what you wanted to talk about.

See: https://jekyllrb.com/docs/configuration/#global-configuration

**Comment by ashmaroli** [comment]
@envygeeks Thats alright, nothing serious, simply wondering aloud

File: site/_config.yml (hunk 1 above)
Comment (parkr) [inline site/_config.yml]: @crispgm Yes! I mean that Jekyll should ignore it when building. See [EntryFilter](https://github.com/jekyll/jekyll/blob/b5ee87b4ca91fefe86c42526fee230a3c0294eca/lib/jekyll/entry_filter.rb#L4-L6).

Comment (crispgm) [inline site/_config.yml]: @parkr sounds that `.gitignore` should be exclude by `EntryFilter`. I will inspect this later.

**Pull request merged**